Translated/Tasks/Limited/2025-07-xx/ls0064390101.csv

```diff
@@ -0,0 +1,5 @@
+Title#0,"""N-Master Cube Acquisition"""
+Summary#0,"""At the Item Lab, exchange Master Fragments<br>for a N-Master Cube<br><br><yellow>※Select \""Upgrade Material Exchange\""<br>\u3000at the Item Lab<c><br>※Can also be acquired through the<br>\u3000\""Seasonal Point Exchange Shop\""<c>"""
```

Comment:
"an N-Master Cube". N is pronounced enn. Also, full stop at the end of this sentence please.